File: CoffeeMachine_Modules/Commande.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CoffeeMachine.Models
{
  public  class Commande
    {
        public int commandeId { get; set; }
        public bool withMug { get; set; }
        public DateTime dateCommande { get; set; }
        public int badgeId { get; set; }
        public int boissonId { get; set; }
        public bool memoeryFlage { get; set; }
        public int sucre { get; set; }
        public virtual  Boisson boisson { get; set; }
        public virtual Badge badge { get; set; }
    }
}

File: CoffeeMachine.FrontEnd/CoffeeMachine/src/app/Dto/Commande.ts
import { Badge } from "./Badge";
import { Boisson } from "./Boisson";

export class Commande {
 /* constructor(badge:Badge){
    this.badge=  badge;
    this.dateCommande=new Date();
  }*/
  constructor(){
    this.dateCommande=new Date();
  }
    commandeId: number;
    withMug:boolean;
    dateCommande:Date;
    badgeId: number;
    boissonId:number;
    memoeryFlage:boolean;
    sucre: number;
    badge:Badge;
    boisson:Boisson;
  }